airline-web/build.sbt

```diff
@@ -31,3 +31,7 @@ resolvers += "scalaz-bintray" at "https://dl.bintray.com/scalaz/releases"
 // Play provides two styles of routers, one expects its actions to be injected, the
 // other, legacy style, accesses its actions statically.
 routesGenerator := InjectedRoutesGenerator
+
+// macOS Big Sur support, this is a work around which might nologer need in play 2.8
+Global / serverConnectionType := ConnectionType.Tcp
```

Comment:
Will need to test and see if this has any impact for windows and linux environment 😊